core/services/webhook/authorizer.go

```diff
@@ -0,0 +1,64 @@
+package webhook
+
+import (
+	uuid "github.com/satori/go.uuid"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"gorm.io/gorm"
+)
+
+type (
```

Comment:
nit - we hashed a while back that the ```type (``` is non-idiomatic - standard is to do just have a the typedef followed immediately by its methods